docs/application/dotnet/guides/nui/nui-components/Notification.md

```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
     notification.Post(3000);
     ```
 
-3. Respond to OnWindowTouch:
+3. Set notfication content:
+
+    ```cs
+    // Sets text in Notification
+    TextLabel text = new TextLabel("This is notification sample.");
+    text.HorizontalAlignment = HorizontalAlignment.Center;
+    text.VerticalAlignment = VerticalAlignment.Center;
+    text.TextColor = Color.Black;
+    text.Size2D = new Size2D(720, 100);
+
+    // Add text label to view
+    view.Add(text);
+
+    // Add view to main Window
+    Window.Instance.Add(view);
+
+    // Post notification for 3000ms
+    notification.Post(3000);
+    ```
+
+4. Respond to OnWindowTouch:

```

Comment:
Just a doubt,  `OnWindowTouch` is an action that triggers a particular event, can we keep this in code styling, like this: 

```suggestion 
4. Respond to `OnWindowTouch`:

Reply:
change done, thank you for your comment